### PR TITLE
fix: tray icon interactions i.e. click to show Pearl

### PR DESCRIPTION
**Pull request by truemiller**
- refactored Tray (toolbar icon)
- combined with tray icons file (removed)
- moved to components folder in an effort to move away from the main script -- it's very messy, monolithic

tested in rc142

### CODE REVIEW

**Comment by truemiller** [comment]
> LGTM, but could you please add a video for both Mac and Windows?

[releasing rc136 draft](https://github.com/valory-xyz/olas-operate-app/actions/runs/10902335125/job/30254111401) now for @oaksprout & @iasonrovis to test

**Comment by truemiller** [comment]
need to fix mac bug where clicking icon shows both content menu & window

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-20.x
+20
```
Comment (truemiller) [inline .nvmrc:1]: fixed as 20.x was invalid

**Review by iasonrovis — APPROVED** [review]

**Review by Tanya-atatakai — APPROVED** [review]
everything is very well explained in the `electron/components/PearlTray.js`, great!

**Pull request merged**